src/main/java/Task.java

```diff
@@ -0,0 +1,39 @@
+public class Task {
+    private String taskDescription;
+    private Boolean isDone;
+    protected TaskType taskType;
+
+    public Task(String taskDescription) {
+        this.taskDescription = taskDescription;
+        this.isDone = false;
+        this.taskType = null;
+    }
+
+    public String getTaskDescription() {
+        return this.taskDescription;
+    }
+
+    public Boolean getTaskStatus() {
+        return this.isDone;
+    }
+
+    public void markAsDone() {
+        this.isDone = true;
+    }
+
+    public void unmarkAsDone() {
+        this.isDone = false;
+    }
+
+    public void printAdd(int todosCount) {
+        System.out.print("");
+    }
```

Comment:
Since this function is for overloading and is void, you can just put as
```suggestion
    public void printAdd(int todosCount) {
        return
    }
```